libsodium-sys/src/crypto_generichash.rs

```diff
@@ -8,6 +8,9 @@ pub const crypto_generichash_KEYBYTES_MAX: usize = crypto_generichash_blake2b_KE
 pub const crypto_generichash_KEYBYTES: usize = crypto_generichash_blake2b_KEYBYTES;
 pub const crypto_generichash_PRIMITIVE: &'static str = "blake2b";
 
+#[allow(non_camel_case_types)]
+pub enum crypto_generichash_state { }
+
```

Comment:
The libsodium docs say "The crypto_generichash_state structure is packed and its length is either 357 or 361 bytes.", I'm not sure how to represent that in rust.

The C code in question is:

```
#pragma pack(push, 1)
typedef CRYPTO_ALIGN(64) struct crypto_generichash_blake2b_state {
    uint64_t h[8];
    uint64_t t[2];
    uint64_t f[2];
    uint8_t  buf[2 * 128];
    size_t   buflen;
    uint8_t  last_node;
} crypto_generichash_blake2b_state;
#pragma pack(pop)
```